src/app/components/work-item-list-entry/work-item-list-entry.component.html

```diff
@@ -46,14 +46,14 @@
                 </div>
               </div>
             </div>
-            <div class="f8-wi__list-description">
+            <div class="f8-wi__list-labels">
               <f8-label [labels]="workItem.relationships?.labels?.data ?
                 workItem.relationships?.labels?.data : []"
                 [truncateAfter]='4'
                 [allowDelete]="false"
                 (onLabelClick)="labelClick($event)"></f8-label>
             </div>
-            <div class="f8-wi__list-description width-100px">
+            <div class="f8-wi__list-status width-100px">
```

Comment:
Why you changed the class names?

Reply:
Becuase they are incorrect. This div contains wi status. Also, protractor tests were throwing warnings as there were 3 `f8-wi__list-description` for each workItem

Reply:
Yes it contains several things such as title, label and status but there are certain common styles set to those divs through that one class. Moreover I choose the name description because it describes the detail of that particular list item. 

You can't change the classnames like this as the styles associated to it are not going to apply now

Reply:
This means we'll have to add IDs to separate description/status/labels. The tests need a unique selector to click on the work item title.

Reply:
Then add Ids but don't change classes

Reply:
Either we add unique classes to description/label/status or we'll have to change the CSS.

Reply:
@SMahil We have multiple workitems and each work item has a description. We cannot use IDs in this case

Reply:
No Dont change CSS. AFA adding ids to label and status, I dont know if you are checking click item on that but they are not clickable

Reply:
in that case add extra class for the item which is going to be clicked

Reply:
I'll add a new class for the clickable div :+1:

Reply:
> The tests need a unique selector to click on the work item title.

@jarifibrahim not really ... you can make use `$$` or `element.all` and filter on those to obtain the unique item. There is also [cssContainingText](http://www.protractortest.org/#/api?view=ProtractorBy.prototype.cssContainingText) api that you can make use of. Have you considered these alternatives?

Reply:
Thanks @sthaha @SMahil. I've updated the PR.